### ISSUE TITLE
check coherency of note.time

### ISSUE THREAD

**Issue by dvribeira** [CARA-ch/ch-emed-epr#37]
For PADV observations, we do not support note.time and, in fact, the aggregator uses the entry time.

Shouldn't we do the same for all note elements in other resources?

**Comment by dvribeira** [comment]
After a brief discussion with S. Spahni, it was decided that all note.time fields shall not be supported by the PMP. This must be changed in the IG.

**Comment by qligier** [comment]
(Except in the medication card, where they are useful)

**Comment by dvribeira** [comment]
Indeed, with the exception of the CHEMEDEPRMedicationStatementCard note.time.
